core/storage/trie/polkadot_trie/polkadot_trie.hpp

```diff
@@ -21,11 +21,14 @@ namespace kagome::storage::trie {
    public:
     using NodePtr = std::shared_ptr<PolkadotNode>;
     using BranchPtr = std::shared_ptr<BranchNode>;
+    using OnDetachCallback = std::function<void(
+        const common::Buffer &key, boost::optional<common::Buffer> &&value)>;
 
     /**
      * Remove all trie entries which key begins with the supplied prefix
```

Comment:
Please comment why this callback is needed.

Reply:
done